### PR TITLE
Nick/netcode example

### PR DESCRIPTION
**Pull request by francobithead**
Reworked Invaders from a P2P Server example to a Dedicated Server example. Also added a login screen and score summary after the game finishes.

### CODE REVIEW

**Comment by michaelc-bitheads** [comment]
It looks like there are two projects being added: Invaders and NetcodeExample.

Are they both being added to the examples?

**Comment by francobithead** [comment]
> It looks like there are two projects being added: Invaders and NetcodeExample.
> 
> Are they both being added to the examples?

No I didn't realize that Nick's tester project was still in the branch. I'll delete it now

File: Invaders/README.md
Comment (michaelc-bitheads) [inline Invaders/README.md:1]: This looks like the README.md for the original example project. It should either be entirely rewritten or deleted.

File: Invaders/Resources/invaders_banner.png
Comment (michaelc-bitheads) [inline Invaders/Resources/invaders_banner.png:1]: This file is used for the README.md and should be deleted.

**Review by michaelc-bitheads — APPROVED** [review]
Just a couple of things to remove. Otherwise I think it should be good 👍

**Review by nick-bithead — APPROVED** [review]

**Pull request merged**